README.md

```diff
@@ -181,18 +179,16 @@ The following platforms are supported by the prebuilt librdkafka binaries:
  * glibc-based Linux x64 (e.g., RedHat, Debian, CentOS, Ubuntu, etc) - without GSSAPI/Kerberos support
  - musl-based Linux 64 (Alpine) - without GSSAPI/Kerberos support
 
-When building your application for Alpine Linux (musl libc) you must pass
+When building your application for Alpine Linux (`musl libc`) you must pass
```

Comment:
musl libc is a project name, not a command, so no fixed-widthing please.

Reply:
You got it!